components/src/form-field/form-field.module.ts

```diff
@@ -0,0 +1,54 @@
+import { CommonModule } from "@angular/common";
+import { NgModule, Pipe, PipeTransform } from "@angular/core";
+
+import { I18nService } from "jslib-common/abstractions/i18n.service";
+
+import { BitInput } from "../input/input.component";
+import { InputModule } from "../input/input.module";
+
+import { BitErrorSummary } from "./error-summary.component";
+import { BitError } from "./error.component";
+import { BitFormFieldComponent } from "./form-field.component";
+import { BitHintComponent } from "./hint.component";
+import { BitLabel } from "./label.directive";
+import { BitPrefix } from "./prefix.directive";
+import { BitSuffix } from "./suffix.directive";
+
+/**
+ * Temporarily duplicate this pipe
```

Comment:
What's going on here? Can't we just import the i18n pipe from jslib?

Reply:
That would create a cyclic dependency. `jslib/angular -> components -> jslib/angular`. The proper solution would be to split up `jslib/angular` but I won't do that until we have migrated to the mono repository.